package.json

```diff
@@ -79,28 +76,32 @@
       "jasmine": true
     }
   },
-  "dependencies": {
+  "peerDependencies": {
+    "carbon-components": "^7.0.0",
     "carbon-icons": "^5.1.2",
+    "react": "^15.3.2",
+    "react-dom": "^15.3.2"
+  },
+  "dependencies": {
     "classnames": "2.2.5",
     "lodash.debounce": "^4.0.8",
-    "react": "^15.3.2",
     "react-addons-css-transition-group": "^15.4.2",
```

Comment:
We can transition, but I'm not sure the transition details. Would be another PR I think